web_monitoring/diffing_server.py

```diff
@@ -57,10 +58,24 @@
 
 client = tornado.httpclient.AsyncHTTPClient()
 
+acao_env_value = \
+    os.environ.get('ACCESS_CONTROL_ALLOW_ORIGIN_HEADER', '-1')
+
 
 class DiffHandler(tornado.web.RequestHandler):
     # subclass must define `differs` attribute
 
+    def set_default_headers(self):
```

Comment:
I think this should also be added to `IndexHandler`, right? Maybe it would be better to create a separate base class that includes these CORS features and have `DiffHandler` and `IndexHandler` inherit from that.